ES/ESdd041.xml

```diff
@@ -71,98 +65,106 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      </msItem>
                      <msItem xml:id="ms_i1.5">
                         <locus from="97va" to="103va"/>
-                        <title>Concluding
-                           rites</title>
+                        <title>Concluding rites</title>
+                     <colophon xml:id="coloph1">
+                        <locus target="#103vb"/>
+                        ለዘአጽሐፎ፡ ለዝንቱ፡ መጽሐፍ፡ <persName ref="PRS14617Zakrestos_"><roleName type="function">አቡነ፡</roleName>  ወ<roleName type="function">መምህርነ፡ </roleName><roleName type="title">አባ፡</roleName>  
```

Comment:
it would be more logical to mark up spaces not as now
`ወ<roleName type="function">መምህርነ፡ </roleName><roleName type="title">አባ፡</roleName> 
`
but as
`ወ<roleName type="function">መምህርነ፡</roleName> <roleName type="title">አባ፡</roleName> `